README.md

```diff
@@ -132,22 +132,22 @@ schema:
 `schema.edge` is a **required** parameter that describes the schema information of the inserted edge. Each edge contains the following three properties:
```

Comment:
`schema.edge` is a **required** parameter that describes the schema information of the inserted edges. Each edge contains the following three properties: